### PR TITLE
jupyter: add pt_BR translation

### PR DESCRIPTION
**Pull request by andrecoco**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/jupyter.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+> Aplicação web para criar e compartilhar documentos que contem código, visualizações e anotações.
+> Usado principalmente para análise de dados, computação científica e aprendizado de máquinas (machine learning).
+> Mais informações: <https://jupyter.org>.
+
+- Inicie um servidor de notebooks Jupyter no diretório atual:
+
+`jupyter notebook`
+
+- Abrir um caderno Jupyter específico:
```
Comment (navarroaxel) [inline pages.pt_BR/common/jupyter.md:11]:
```suggestion
- Abre um caderno Jupyter específico:
```

File: pages.pt_BR/common/jupyter.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# jupyter
+
+> Aplicação web para criar e compartilhar documentos que contem código, visualizações e anotações.
+> Usado principalmente para análise de dados, computação científica e aprendizado de máquinas (machine learning).
+> Mais informações: <https://jupyter.org>.
+
+- Inicie um servidor de notebooks Jupyter no diretório atual:
```
Comment (navarroaxel) [inline pages.pt_BR/common/jupyter.md:7]:
```suggestion
- Inicia um servidor de notebooks Jupyter no diretório atual:
```

File: pages.pt_BR/common/jupyter.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`jupyter notebook`
+
+- Abrir um caderno Jupyter específico:
+
+`jupyter notebook {{exemplo.ipynb}}`
+
+- Exportar um caderno Jupyter específico para outro formato:
```
Comment (navarroaxel) [inline pages.pt_BR/common/jupyter.md:15]:
```suggestion
- Exporta um caderno Jupyter específico para outro formato:
```

File: pages.pt_BR/common/jupyter.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`jupyter nbconvert --to {{html|markdown|pdf|script}} {{exemplo.ipynb}}`
+
+- Inicie um servidor em uma porta específica:
```
Comment (navarroaxel) [inline pages.pt_BR/common/jupyter.md:19]:
```suggestion
- Inicia um servidor em uma porta específica:
```

File: pages.pt_BR/common/jupyter.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`jupyter notebook --port={{porta}}`
+
+- Lista de servidores de notebooks atualmente em funcionamento:
+
+`jupyter notebook list`
+
+- Pare o servidor atualmente em funcionamento:
```
Comment (navarroaxel) [inline pages.pt_BR/common/jupyter.md:27]:
```suggestion
- Para o servidor atualmente em funcionamento:
```

File: pages.pt_BR/common/jupyter.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+`jupyter notebook stop`
+
+- Inicie o JupyterLab, se instalado, no diretório atual:
```
Comment (navarroaxel) [inline pages.pt_BR/common/jupyter.md:31]:
```suggestion
- Inicia o JupyterLab, se instalado, no diretório atual:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @andrecoco

**Pull request merged**